ocs_ci/deployment/deployment.py

```
@@ -3821,14 +3807,8 @@ class MultiClusterDROperatorsDeploy(object):
 
     """
 
-    def __init__(self, dr_conf):
-        self.meta_map = {
-            "awss3": self.s3_meta_obj_store,
-            "mcg": self.mcg_meta_obj_store,
-        }
-        # Default to s3 for metadata store
-        self.meta_obj_store = dr_conf.get("dr_metadata_store", "awss3")
-        self.meta_obj = self.meta_map[self.meta_obj_store]()
+    def __init__(self):
```

Ruff (0.16.2)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)